### ISSUE TITLE
Generates import statements for binaries

### ISSUE THREAD

**Issue by mattouille** [princjef/gomarkdoc#67]
When I run `gomarkdoc ./... > doc.md` on a package whose name is `main` I was expecting that it would skip creating the import statement or that it would contain a `go install` statement instead.

**Comment by princjef** [comment]
You're right that an import statement is a bit nonsensical for a `main` package. I'm a bit hesitant to change the default behavior here at this point, but I've made [changes](https://github.com/princjef/gomarkdoc/commit/80124ce84a76d3eafedae0c6825d02e50dc141cd) to split the import into its own template and added the import path as a standalone field. With these two things combined, you can achieve the behavior you want by simply adding this to your `.gomarkdoc.yml`:

```yml
template:
  import: |
    {{- if eq .Name "main" -}}
      {{- codeBlock "" (print "go install " .ImportPath) -}}
    {{- else -}}
      {{- codeBlock "go" .Import -}}
    {{- end -}}
```

**Comment by princjef** [comment]
This will be enabled with the next release, which should be cut soon

**Comment by princjef** [comment]
This is available in v0.4.0
